src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+        int taskDescriptionEndIndex;
+        
+        //find start of task description
+        taskDescriptionStartIndex = userInput.indexOf(" ");
+        taskDescriptionEndIndex = userInput.indexOf("/at");
+        //assign task's information
+        description = userInput.substring(taskDescriptionStartIndex + 1, taskDescriptionEndIndex).trim();
+        String at = userInput.substring(taskDescriptionEndIndex + 3).trim();//+3 to exclude "/at"
+        taskType = TaskType.EVENT;
+        //create and assign information to new event
+        Event event = new Event(description,taskType,at);
+        tasks[listSize++] = event;
+    }
+    
+    //add deadline to list
+    private static void addDeadline(String userInput) {
+        TaskType taskType;
+        String description;
+        
+        //start and end index for description
+        int taskDescriptionStartIndex;
+        int taskDescriptionEndIndex;
+        
+        //find start of task description
+        taskDescriptionStartIndex = userInput.indexOf(" ");
+        taskDescriptionEndIndex = userInput.indexOf("/by");
+        //assign task's information
+        description = userInput.substring(taskDescriptionStartIndex + 1, taskDescriptionEndIndex).trim();
+        String by = userInput.substring(taskDescriptionEndIndex + 3).trim();//+3 to exclude "/by"
+        taskType = TaskType.DEADLINE;
+        //create and assign information to new deadline
+        Deadline deadline = new Deadline(description,taskType,by);
+        tasks[listSize++] = deadline;
+    }
+    
+    //add todo to list
+    private static void addTodo(String userInput) {
+        TaskType taskType;
+        String description;
+        
+        //start index for description
+        int taskDescriptionStartIndex;
+        
+        //find start of task description
+        taskDescriptionStartIndex = userInput.indexOf(" ");
+        //assign task's information
+        description = userInput.substring(taskDescriptionStartIndex + 1);
+        taskType = TaskType.TODO;
+        //create and assign information to new ToDo
+        ToDo toDo = new ToDo(description,taskType);
+        tasks[listSize++] = toDo;
+    }
+    
+    //display current list
+    public static void displayList(){
+        System.out.print(
+                "____________________________________________________________\n"+
+                " Here are the tasks in your list:\n"
+        );
+        //print out all list's items
+        for (int i = 0;i < listSize;i++){
+            tasks[i].displayListPerTask(i);
+        }
+        System.out.print(
+                "____________________________________________________________\n"
+        );
+    }
+    
+    //handle user input commands
+    private static void handleCommand() {
+        //activate scanner
+        Scanner input = new Scanner(System.in);
+        String userInput; //user input
+        
+        //scan user's input
+        userInput = input.nextLine();
```

Comment:
Hello again! perhaps, you can make this in line 115 such that it is not redundant like "String userInput = input.nextLine(); and also can comment it as //scan user input